macros/edr/data_monitoring/monitors/monitors.sql

```diff
@@ -4,6 +4,7 @@
       'column_any_type': ['null_count', 'null_percent', 'not_null_percent'],
       'column_string': ['min_length', 'max_length', 'average_length', 'missing_count', 'missing_percent', 'not_missing_percent'],
       'column_numeric': ['min', 'max', 'zero_count', 'zero_percent', 'not_zero_percent', 'average', 'standard_deviation', 'variance', 'sum']
```

Comment:
You're missing a semicolon here

Reply:
This is why the tests fail

Reply:
@noaKurman Ah when I merged main into my branch, I missed it when resolving the conflicts. It is ready now!